scripts/binary/binary-entry-point-source.js

```diff
@@ -0,0 +1,21 @@
+const Module = require('module')
+const path = require('path')
+
+process.env.CYPRESS_INTERNAL_ENV = process.env.CYPRESS_INTERNAL_ENV || 'production'
+try {
+  require('./node_modules/bytenode/lib/index.js')
```

Comment:
```suggestion
  require('bytenode/lib/index.js')
```

Reply:
I'm actually trying to purposefully reference the exact file here. I don't anyone to be able to use any kind of module loading weirdness to reference a different file (may not be possible, but just in case).